sources/deny.toml

```diff
@@ -66,6 +66,9 @@ skip-tree = [
     # aws-smithy-experimental brings a set of new dependencies that
     # the aws-sdk doesn't support
     { name = "aws-smithy-experimental", version = "=0.1" },
+    # zbus introduces newer versions of crates that are either
+    # incompatible or way ahead from the versions used in other crates
+    { name = "zbus", version = "=5.11" }
```

Comment:
I'd like to try to resolve this now if we can or else document what's keeping us on older crates.